features/lwipstack/mbed_lib.json

```diff
@@ -88,6 +88,14 @@
             "help": "TCP sender buffer space (bytes). Current default (used if null here) is set to (4 * TCP_MSS) in opt.h, unless overridden by target Ethernet drivers.",
             "value": null
         },
+        "tcp-maxrtx": {
+            "help": "Maximum number of retransmissions of data segments.",
+            "value": 6
+        },
+        "tcp-synmaxrtx": {
+            "help": "Maximum number of retransmissions of SYN segments.",
```

Comment:
If you're allowing `null` to get at lwIP's default, it's helpful to specify what the default is - all the other entries here do.

Reply:
Agree.

I'll update.